```
mitkSceneIO.cpp:50:10: fatal error: QtConcurrent\qtconcurrentrun.h: No such file or directory
 #include <QtConcurrent\qtconcurrentrun.h>
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

--- a/Modules/SceneSerialization/CMakeLists.txt
+++ b/Modules/SceneSerialization/CMakeLists.txt
@@ -1,6 +1,6 @@
 MITK_CREATE_MODULE(
   DEPENDS MitkSceneSerializationBase MitkLogging
-  PACKAGE_DEPENDS PUBLIC Boost Poco|Zip Qt5|Core
+  PACKAGE_DEPENDS PUBLIC Boost Poco|Zip Qt5|Core+Widgets
   WARNINGS_AS_ERRORS
 )
 

--- a/Modules/SceneSerialization/src/mitkSceneIO.cpp
+++ b/Modules/SceneSerialization/src/mitkSceneIO.cpp
@@ -45,6 +45,10 @@ See LICENSE.txt or http://www.mitk.org for details.
 
 #include "itksys/SystemTools.hxx"
 
+#include <QApplication>
+#include <QFuture>
+#include <QtConcurrent/QtConcurrent>
+
 #include "AutoplanLogging.h"
 
 mitk::SceneIO::SceneIO()
@@ -201,7 +205,15 @@ mitk::DataStorage::Pointer mitk::SceneIO::LoadScene( const std::string& filename
   Poco::Zip::Decompress unzipper( file, Poco::Path( m_WorkingDirectory ) );
   unzipper.EError += Poco::Delegate<SceneIO, std::pair<const Poco::Zip::ZipLocalFileHeader, const std::string> >(this, &SceneIO::OnUnzipError);
   unzipper.EOk    += Poco::Delegate<SceneIO, std::pair<const Poco::Zip::ZipLocalFileHeader, const Poco::Path> >(this, &SceneIO::OnUnzipOk);
-  unzipper.decompressAllFiles();
+
+  QFuture<void> future = QtConcurrent::run([this, &unzipper]() {
+    unzipper.decompressAllFiles();
+  });
+  
+  while (!future.isFinished()) {
+    qApp->processEvents();
+  }
+
   unzipper.EError -= Poco::Delegate<SceneIO, std::pair<const Poco::Zip::ZipLocalFileHeader, const std::string> >(this, &SceneIO::OnUnzipError);
   unzipper.EOk    -= Poco::Delegate<SceneIO, std::pair<const Poco::Zip::ZipLocalFileHeader, const Poco::Path> >(this, &SceneIO::OnUnzipOk);
 

